yt/geometry/_selection_routines/selector_object.pxi

```diff
@@ -196,7 +196,7 @@ cdef class SelectorObject:
         cdef np.float64_t dds[3]
         cdef np.float64_t pos[3]
         cdef int ci, cj, ck
-        cdef int nr = (1 << (visitor.oref - 1))
+        cdef int nr = (self.oref * 0.5)
```

Comment:
Are you sure the following code is correct if `oref` if odd, e.g. `self.oref = 3`?

Reply:
I suspect it wouldn't work, but before this PR the number of zones (which `oref` is specifying here) was always even, so it won't break anything (and I can't test any changes to make it compatible).

Separately, it should be visitor.oref, not self.oref, so I will change that.